extension/webviews/components/ViewerTopBar.svelte

```diff
@@ -1,51 +1,73 @@
 <script type="ts">
   import Button from './Button.svelte';
   import { Page } from '../enums';
-  import { currentProject, page } from '../state/store';
+  import { currentOrganization, currentProject, currentUser, page } from '../state/store';
   import DropdownMenu from './DropdownMenu.svelte';
+  import type { DropdownMenuOption, User } from '../types';
 
-  export let username: string;
   export let reload: () => void;
   export let logout: () => void;
+  let organizationName: string | undefined;
+  let projectName: string | undefined;
+  let contextMenuDropdownOptions: DropdownMenuOption[];
+
+  $: organizationName = $currentOrganization?.name ? $currentOrganization?.name.split(' ').join('-').toLowerCase() : '';
+  $: projectName = $currentProject?.name ? $currentProject?.name.split(' ').join('-').toLowerCase() : '';
+  $: contextMenuDropdownOptions = getContextMenuOptions($currentUser, $page);
 
   const switchToInvitePage = () => {
     $page = Page.InviteUser;
   };
 
-  $: showInviteButton = $page === Page.ThreadsViewer || $page === Page.ReplyViewer;
-
-  $: contextMenuDropdownOptions = [
-    {
-      key: 'logout',
-      label: 'Logout',
-      handler: logout,
-    },
-    ...(showInviteButton
-      ? [
-          {
-            key: 'invite',
-            label: 'Invite',
-            handler: switchToInvitePage,
-          },
-        ]
-      : []),
-  ];
+  const getContextMenuOptions = (currentUser: User | null, page: Page) => {
+    const options: DropdownMenuOption[] = [];
+
+    if (currentUser) {
+      options.push({
+        key: 'logout',
+        label: 'Logout',
+        handler: logout,
+      });
+    }
+
+    if (page === Page.ThreadsViewer || page === Page.ReplyViewer) {
```

Comment:
we need to add the currentUser logic here as well otherwise it doesn't make much sense to invite someone when you are not logged in.

Reply:
Will fix